lib/src/rules/avoid_types_on_closure_parameters.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+library linter.src.rules.avoid_annotating_types_on_function_expressions;
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+import 'package:linter/src/analyzer.dart';
+
+const _desc = r'Avoid annotating types on function expression parameters.';
```

Comment:
No, it is just that we are detecting it in FunctionExpression that are not inside of a FunctionDeclaration (an anonymous function, that are known as "lambdas" for me) even the name of the rule changed to: Avoid types on closure parameters, so I feel uncomfortable about not having a name for the concept of an anonymous function, I think if its closure or lambda or whatever it would be better than "function expression".

So what is the correct name for anonymous functions in Dart?